Repository/RevisionRepository.php

```diff
@@ -558,30 +558,23 @@ public function deleteRevisions(ContentType $contentType = null)
         }
     }
 
-    /**
-     * @param ContentType $contentType
-     * @param \DateTime $until
-     * @param string $by
-     * @param bool $force
-     *
-     * @param bool $id
-     * @return int affected rows
-     * @throws \Exception
-     */
-    public function lockRevisions(ContentType $contentType, \DateTime $until, $by, $force = false, $id = false)
+    public function lockRevisions(?ContentType $contentType, \DateTime $until, $by, $force = false, $id = false): int
     {
-        $params = ['by' => $by, 'until' => $until, 'content_type' => $contentType];
+        $params = ['by' => $by, 'until' => $until];
 
-        /** @var QueryBuilder $qbSelect */
         $qbSelect = $this->createQueryBuilder('s');
         $qbSelect
             ->select('s.id')
-            ->andWhere($qbSelect->expr()->eq('s.contentType', ':content_type'))
             ->andWhere($qbSelect->expr()->isNull('s.endTime'))
             ->andWhere($qbSelect->expr()->eq('s.deleted', $qbSelect->expr()->literal(false)))
             ->andWhere($qbSelect->expr()->eq('s.draft', $qbSelect->expr()->literal(false)))
         ;
 
+        if (null !== $contentType) {
+            $params['content_type'] = $contentType;
```

Comment:
you can the QueryBuilder setParameter() function multiple times, that way you don't need to keep the params array.